manifests/opsmanager.pp

```diff
@@ -0,0 +1,59 @@
+# This installs Ops Manager. See README.md for more info.
+
+class mongodb::opsmanager (
+  String[1] $user                                = $mongodb::params::opsmanager_user,
+  String[1] $group                               = $mongodb::params::opsmanager_group,
+  Variant[Boolean, String[1]] $ensure            = $mongodb::params::opsmanager_ensure,
+  String[1] $package_name                        = $mongodb::params::opsmanager_package_name,
+  Boolean $package_ensure                        = $mongodb::params::opsmanager_package_ensure,
+  Boolean $service_enable                        = $mongodb::params::opsmanager_service_enable,
+  Boolean $service_manage                        = $mongodb::params::opsmanager_service_manage,
+  String[1] $service_name                        = $mongodb::params::opsmanager_service_name,
+  String[1] $download_url                        = $mongodb::params::opsmanager_download_url,
+  String[1] $mongo_uri                           = $mongodb::params::opsmanager_mongo_uri,
+  String[1] $hostname                            = $mongodb::params::opsmanager_hostname,
+  String[1] $protocol                            = $mongodb::params::opsmanager_protocol,
```

Comment:
This is actually unused so I'd leave it out.

Reply:
This stuff is used to build the opsmanager_url, reason being that the defaults for hostname, protocol, and port make it so that a user doesn't need to input the opsmanager_url at all if they don't need to. If we don't want to support this default url behavior I'm cool with getting rid of the defaults and explicitly requiring a URL to be included.

Reply:
As I undestand it, that's not how the interpolation works. `opsmanager_url` is taken from `params.pp` which interpolates it with local variables. It doesn't look at variables from a completely different scope.

If you're sure this works, this needs a test but I'd prefer a single `$opsmanager_url`